This patch is needed for GTK to make sure that the port finds the right OpenGL library
--- configure.orig
+++ configure
@@ -19465,48 +19465,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
 
-SEARCH_INCLUDE="\
-    /usr/local/include        \
-    /usr/local/X11/include    \
-    /usr/local/include/X11    \
-    /usr/local/X11R7/include  \
-    /usr/local/X11R6/include  \
-    /usr/local/include/X11R7  \
-    /usr/local/include/X11R6  \
-                              \
-    /usr/Motif-2.1/include    \
-    /usr/Motif-1.2/include    \
-    /usr/include/Motif1.2     \
-                              \
-    /usr/dt/include           \
-    /usr/openwin/include      \
-                              \
-    /usr/include/Xm           \
-                              \
-    /usr/X11R7/include        \
-    /usr/X11R6/include        \
-    /usr/X11R6.4/include      \
-                              \
-    /usr/include/X11R7        \
-    /usr/include/X11R6        \
-                              \
-    /usr/X11/include          \
-    /usr/include/X11          \
-                              \
-    /usr/XFree86/include/X11  \
-    /usr/pkg/include          \
-                              \
-    /usr/local/X1R5/include  \
-    /usr/local/include/X11R5  \
-    /usr/X11R5/include        \
-    /usr/include/X11R5        \
-                              \
-    /usr/local/X11R4/include  \
-    /usr/local/include/X11R4  \
-    /usr/X11R4/include        \
-    /usr/include/X11R4        \
-                              \
-    /usr/openwin/share/include"
+SEARCH_INCLUDE="@@PREFIX@@/include"
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for libraries directories" >&5
 $as_echo_n "checking for libraries directories... " >&6; }
